scripts/compile/jsc.sh

```diff
@@ -50,6 +50,19 @@ else
     BUILD_TYPE_FLAGS="-DDEBUG_FISSION=OFF"
 fi
 
+if [[ "$ARCH_NAME" = "i686" ]]
+then
+    JSC_FEATURE_FLAGS=" \
+      -DENABLE_JIT=OFF \
```

Comment:
why do we still diable JIT in x86 builds?

Reply:
After new bytecode format change from WebKit, 32bits platform JIT is not supported.
arm32 support is contributed by WebKit community (from Igalia)

There will be compile error if enable JIT for x86.